infrastructure/resource_group_level/main.bicep

```diff
@@ -0,0 +1,138 @@
+param location string  = resourceGroup().location
+param environment string = 'feature'
+
+param appServicePlanName string = 'feature'
+param appServicePlanSku string = 'p2v2'
+param appServicePlanKind string = 'linux'
+
+@minLength(5)
+@maxLength(50)
+@description('Provide a globally unique name of your Azure Container Registry')
+param acrName string = 'tfsfeatureacr${uniqueString(resourceGroup().id)}'
+@description('Provide a tier of your Azure Container Registry.')
+// Dev uses 'Standard'
+param acrSku string = 'Basic'
+
+
+// Dev uses 172.16.4x.xx
+// Demo (legacy?) uses 172.16.6x.xx
```

Comment:
Can I kill everything to do with Demo?
Not that Test & Staging seem to be overlapping too. Is this expected? @IgnatG